pkgs/tools/admin/daemontools/default.nix

```diff
@@ -32,8 +40,15 @@ stdenv.mkDerivation rec {
     for cmd in $(cat package/commands); do
       install -Dm755 "command/$cmd" "$out/bin/$cmd"
     done
+
+    tar -xz --strip-components=2 -f ${man-pages}
+    install -v -Dm644 -t $man/share/man/man8 daemontools-man/*.8
```

Comment:
```suggestion
    installManPage daemontools-man/*.8
```

Reply:
Thanks. I updated the commit with you suggestion.